### DISCUSSION TITLE
Help coding a conditional constraint in PuLP

### DISCUSSION THREAD

**Discussion by 0blue00** [coin-or/pulp#725] [Q&A] [answered]
I am trying to code a problem in PuLP where I have 1 list of binary variables (lets call it L) and 1 binary variable (lets call it b).
I want to assign b a value of 0 if sum of values in L is 0, else b should be assigned value 1.
I am trying to add this as a constraint but unable to handle the conditional part.

Any inputs on this are welcome. TIA!

**Comment by pchtsp** [comment, marked as answer]
Hello! Check 7.2 in this manual: https://download.aimms.com/aimms/download/manuals/AIMMS3OM_IntegerProgrammingTricks.pdf
